packages/block-library/src/comments/edit/comments-legacy.js

```diff
@@ -57,11 +57,10 @@ export default function CommentsLegacy( {
 			<div { ...blockProps }>
 				<Warning actions={ actions }>
 					{ __(
-						"Comments block: You're currently using this block in legacy mode. " +
-							'The following is just a placeholder, not a real comment. ' +
-							'The final styling may differ because it also depends on the current theme. ' +
-							'For better compatibility with the Block Editor, ' +
-							'please consider switching the block to its editable mode.'
+						"Comments block: You're currently using the legacy version. " +
+							'The following is just a placeholder -- it might look different on the frontend. ' +
+							'For a better representation and more customization options, ' +
```

Comment:
Two benefits:
- It's a more faithful visual representation of the outcome on the frontend.
- It can be edited: not just customizing fonts etc, but also re-arranging Comments Title/Author/Content blocks, adding other blocks, etc. Not sure how to best call that, so I went with "customization" for all of them. Any better ideas? How about

```suggestion
							'For a better representation and more composition and customization options, ' +
```